cmd/fontman/commands/uninstall.go

```diff
@@ -16,12 +23,15 @@ func onUninstall(c *cli.Context) error {
 // Constructs the 'uninstall' subcommand.
 func RegisterUninstall() *cli.Command {
 	var style string
+	var ex_style string
```

Comment:
Same thing as above, I think we should stick with camelCase if possible (I prefer pascal_case, but Go likes camelCase 🤷‍♂️)